src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    int area();
```

Comment:
1. areas can be double
2. better name. method name should sound like an action
```suggestion
    double calculateArea();
```